lib/commands/deployApi/deployApi.js

```diff
@@ -11,6 +11,7 @@ var questions = [
   { name: 'organization', message: 'Organization?' },
   { name: 'username', message: 'User Id?' },
   { name: 'password', message: 'Password?', type: 'password' },
+  { name: 'token', message: 'token?'},
```

Comment:
@inafis - Thanks for the PR
Can you add the space between `'token?'` and `}` 
something like this
`{ name: 'token', message: 'token?' },`

Because of this lint error, the Travis build [failed](https://travis-ci.org/github/anil614sagar/openapi2apigee/builds/759378598)

Reply:
Whoops - I did notice this and forgot to commit and push this change. Done.